juniper_official/interface/check-interface-in-out-errors-traffic-state-flaps.rule

```diff
@@ -66,6 +66,9 @@ healthbot {
                 }
                 type string;
                 description "Interface admin status";
+                enumerate __default__ as -1;
```

Comment:
Isn't admin and open status have up and down, instead of true and false????

Reply:
yes, I have made the changes as Up and down